tests/validation/tests/v3_api/common.py

```diff
@@ -489,6 +503,8 @@ def validate_workload(p_client, workload, type, ns_name, pod_count=1,
         assert wl_result["status"]["currentNumberScheduled"] == len(pods)
     if type == "cronJob":
         assert len(wl_result["status"]["active"]) >= len(pods)
+    if type == "job":
+        assert wl_result["status"]["succeeded"] == len(pods)
 
```

Comment:
For the workloads of type job the status succeeded equals the number of pods that are present for that workload.